module/VuFind/src/VuFind/RecordDriver/Search2Default.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Default model for Search2 records -- used when a more specific model based on
+ * the record_format field cannot be found.
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2010.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  RecordDrivers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Samuli Sillanpää <samuli.sillanpaa@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:record_drivers Wiki
+ */
+namespace VuFind\RecordDriver;
+
+/**
+ * Default model for Search2 records -- used when a more specific model based on
+ * the record_format field cannot be found.
+ *
+ * This should be used as the base class for all Solr-based record models.
+ *
+ * @category VuFind
+ * @package  RecordDrivers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @author   Samuli Sillanpää <samuli.sillanpaa@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:record_drivers Wiki
+ *
+ * @SuppressWarnings(PHPMD.ExcessivePublicCount)
+ */
+class Search2Default extends SolrDefault
```

Comment:
I think this class should be registered in VuFind\RecordDriver\PluginManager -- it may be working because of default behavior, but it would be best to be more explicit.

Reply:
Yes, updated. Maybe it works because `RecordCollectionFactory` uses `getSearch2Record`.